### PR TITLE
fix

### PR DESCRIPTION
**Pull request by raymondng1**
1. Run App 
2. Log In as your **Admin User** 
3. Upload an image in the create climbing route component
4. Check the database for the updated associations with the climbingRouteId(many relationship) and the userId will be associated with the uploaded image, as well as a new climbing route being created as pending

### CODE REVIEW

File: server/api/routeImages.js [hunk 1]
```diff
@@ -35,13 +35,25 @@ router.post('/', (req, res, next) => {
         return res.status(500).send(err);
       }
       //if no error,add the image name and send the file name and path back to the client
-      RouteImage.create({
-        fileName: file.name,
-        filePath: `/uploads/${file.name}`,
-        userId: req.user.id,
-      }).then(() => {
-        res.send({ fileName: file.name, filePath: `/uploads/${file.name}` });
+      ClimbingRoute.create({
```
Comment (nries1) [inline server/api/routeImages.js:41]: We actually don't want to create a new climbing route every time some1 uploads a video. We just need to associate the new image with a climbing route and id. So you're creating a new entry in the  route image table and giving it a value for climbingRouteId and userId

Comment (nries1) [reply server/api/routeImages.js:41]: Also shouldn't the file path be `./assets/uploads/${file.name}`  ?

Comment (raymondng1) [reply server/api/routeImages.js:41]: 1. Yeah i will make the fix for the climbing route, thanks for the clarification !

2. The file path for the image when it is uploaded is **./public/uploads/${file.name}**  

Essentially would you want the image to be uploaded into the assets section as it's own separate directory instead, like  **./public/assets/uploads/${file.name}**   ?

Comment (raymondng1) [reply server/api/routeImages.js:41]: @nries1  or did you mean the file path/url to be: **`./public/uploads/${file.name}`**

**Review by nries1 — CHANGES_REQUESTED** [review]
@raymondng1 . I'm so sorry for my comment earlier. I realize what you were doing now. You want to create a new route when someone posts an image so that you can attach the img to that route right?

That makes sense. The only problem is that you'll need more info about the route than just the image in order to create a route. So if you wanna create a route on image upload you'll need to collect some other info first. 

One other note: I ran this branch on my local machine and I wasn't able to save an image to the db. I needed to change the syntax for saving the image to this:

```
file.mv(
    path.join(
      __dirname,
      '../..',
      '/public',
      `/uploads/${file.name.split(' ').join('-')}`
    )
```

**Review by nries1 — APPROVED** [review]
NICE

**Pull request merged**